wg-serving/charter.md

```diff
@@ -0,0 +1,83 @@
+# WG Serving Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+Discuss and enhance the support for AI/ML inference workloads in Kubernetes.
+
+
+### In scope
+
+- Provide concrete input to other SIGs and WG around needs of inference
+  workloads.
+- Gather requirements for serving workloads (inference primarily, but benefiting
+  other non-batch use cases where possible) that have broad community alignment
+  from practitioners, distros, and vendors.
+- Directly improve key kubernetes workload controllers when used with
+  accelerators and the most common inference serving frameworks and model
+  servers.
+- Partner with existing ecosystem projects like kServe, Seldon, Kaito, and
+  others to identify, extract, or implement common shared problems (like Kueue
+  abstracted deferred scheduling for multiple batch frameworks).
+- Explore new projects that improve orchestration, scaling, and load balancing
+  of inference workloads and compose well with other workloads on Kubernetes
+
+### Out of scope
```

Comment:
We need to spend more time calling out out of scope items

Reply:
workflows was a good call. Ease of integrating with plugins may be in scope?